README.md

```diff
@@ -62,3 +62,70 @@ or start on the [landing page](http://localhost:8000/)
 14. README.md should contain a link to your personal docker hub repository win an app image
 15. README.md should contain instructions on how to access the application via a browser.
 16. Create PR with your changes and attach it for validation on a platform
+
+# Todo App Deployment Instructions
+
+## Running MySQL Container with Volume
+
+## MySQL Container Setup
+
+To run the MySQL container with persistent storage, you need to create a volume and then run the container with that volume attached.
+
+
+### Step 1: Create a Volume
+
+This command creates a new Docker volume named `mysql-data`:
+
+
+    docker volume create mysql-data
+
+
+### Step 2: Run MySQL Container
+Use this command to start the MySQL container with the volume attached:
+
+    docker run --name mysql-container -v mysql-data:/var/lib/mysql -e MYSQL_ROOT_PASSWORD=1234 -e MYSQL_DATABASE=app_db -e MYSQL_USER=app_user -e MYSQL_PASSWORD=1234 -p 3306:3306 -d mysql-local:1.0.0
```

Comment:
Those env's declared inside Dockerfile, no reason to override them during execution run command.
As Nazar suggested in the previous review - add build image instructions and use the build image in the run command.

Reply:
I got it !!